### ISSUE TITLE
Missing Poison on phoenix 1.4

### ISSUE THREAD

**Issue by sigu** [boydm/phoenix_integration#24]
### Compilation time
When compiling the dependencies I got the following error
```bash
==> phoenix_integration
Compiling 3 files (.ex)
warning: function Poison.decode!/1 is undefined (module Poison is not available)
Found at 2 locations:
  lib/phoenix_integration/assertions.ex:388
  lib/phoenix_integration/assertions.ex:411
```
### running tests
Later on when doing assertions for `json` response I got the error shown below
```bash
** (UndefinedFunctionError) function Poison.decode!/1 is undefined (module Poison is not available)
```
Will this switch to using Jason or can `Poison` be added as a dependency of this project. Let me know of your preferred solution so I can help :smile: 

## proposed solution that I used
Added the `poison` library to my mix file for use on test environment only and now things are working

**Comment by boydm** [comment]
I should probably move this from poison to jason... Making a note of that here

**Comment by boydm** [comment]
Just published version 0.6 to hex. This moves away from Poison and now uses Jason, which is what Phoenix 1.4 uses.
